src/views/charts/LaunchableMarketChart.tsx

```diff
@@ -265,7 +266,14 @@ export const LaunchableMarketChart = ({
   );
 };
 
-const $LaunchableMarketChartContainer = tw.div`flex h-fit w-[25rem] flex-col gap-1 rounded-[1rem] border-[length:--border-width] border-color-border p-1.5 [border-style:solid]`;
+const $LaunchableMarketChartContainer = styled.div`
+  ${tw`flex h-fit w-[25rem] flex-col gap-1 rounded-[1rem] border-[length:--border-width] border-color-border p-1.5 [border-style:solid]`}
```

Comment:
objectively, maybe bad code, but it brings me joy

Reply:
If it wrapped I would've pulled out the props 😆